content/en/docs/concepts/architecture/nodes.md

```diff
@@ -215,34 +215,15 @@ of the Node resource. For example, the following JSON structure describes a heal
 ]
 ```
 
-If the `status` of the Ready condition remains `Unknown` or `False` for longer
-than the `pod-eviction-timeout` (an argument passed to the
-{{< glossary_tooltip text="kube-controller-manager" term_id="kube-controller-manager"
->}}), then the [node controller](#node-controller) triggers
-{{< glossary_tooltip text="API-initiated eviction" term_id="api-eviction" >}}
-for all Pods assigned to that node. The default eviction timeout duration is
-**five minutes**.
-In some cases when the node is unreachable, the API server is unable to communicate
-with the kubelet on the node. The decision to delete the pods cannot be communicated to
-the kubelet until communication with the API server is re-established. In the meantime,
-the pods that are scheduled for deletion may continue to run on the partitioned node.
-
-The node controller does not force delete pods until it is confirmed that they have stopped
```

Comment:
I couldn't see this behavior in the code or in testing.  To verify:

1. I stopped kubelet on a node with running pods
2. The node went NotReady and was tainted
3. After the taint grace period expired, the pods were evicted.
4. The pods were still running on the node with stopped kubelet and replacements were created and running on a different node.